jdi-light-material-ui/src/main/java/com/epam/jdi/light/material/asserts/layout/GridListAssert.java

```diff
@@ -7,11 +7,14 @@
 import com.epam.jdi.light.material.elements.layout.GridList;
 import org.hamcrest.Matchers;
 
+/**
+ * Assertions for {@link GridList}.
+ */
 public class GridListAssert extends UISelectAssert<GridListAssert, GridList> implements HasAssert<GridListAssert> {
 
     @JDIAction("Assert that '{name}' element {0} has title '{1}'")
     public GridListAssert title(int elNum, String reqTitle) {
-        jdiAssert(element().getTitle(elNum), Matchers.is(reqTitle));
+        jdiAssert(element().title(elNum), Matchers.is(reqTitle));
```

Comment:
Maybe it's better not to abbreviate variable names `elNum` and `reqTitle`?
The same goes for `altImgName()`.